src/components/molecules/Sidebar/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./Sidebar";
```

Comment:
[NB, nit] As above, prefer named exports

Reply:
nit/stylistic that we haven't established our stance on yet, resolving this comment